project_tests.sh

```diff
@@ -15,4 +15,4 @@ echo "Reindexing api-dummy"
 bin/ci-reindex
 
 echo "Reindexing RDS articles"
-bin/reindex-rds
+bin/ci-reindex-rds
```

Comment:
Can we make changes to `bin/reindex-rds` to not have to support 2 files?

Reply:
With some further work possibly, but the reason they need to be separate now is because the mechanisms for starting the workers and queue watchers is different between the CI and Production environments, and in this case I discovered that the CI script is being used by prod which could result in an edge case of content being inserted into the wrong index, similar to what I'm able to see locally.
As such, I split the scripts out for now and cleaned the production one up to not start the processes manually and instead rely on the fact that `systemd` has already started them.

Reply:
+1 , these scripts are already spaghetti and trading a little duplicated code for more clarity isn't so bad.